### ISSUE TITLE
Database user is not allowed to re-create the database

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/ofn-install#819]
## Description

A change in the way we use the postgresql role means that the CREATEDB privilege is not passed on to the user creation. We can't drop and create the database. That is useful when importing another database, for example when switching production servers.

## Expected Behavior

```
DISABLE_DATABASE_ENVIRONMENT_CHECK=1 I_AM_SURE=1 bundle exec rails db:drop db:create
# database created
```

## Actual Behaviour

```
DISABLE_DATABASE_ENVIRONMENT_CHECK=1 I_AM_SURE=1 bundle exec rails db:drop db:create
# error on db:create
```

## Steps to Reproduce

1. Fresh server, for example with `vagrant destroy; vagrant up`.
2. Provision and deploy.
3. Try to re-create the database.

## Your Environment

* Version used:
* Browser name and version:
* Operating System and version (desktop or mobile):

## Possible Fix

We added some workarounds in ebd62cd0a7462b0afecf4ca0a090066444b8fa08 which could now be removed if we upgrade the postgresql role.

**Comment by jibees** [comment]
Could we test it with a local setup (and only local...)?

**Comment by mkllnk** [comment]
Yes, we can totally test it locally. That's what I mean in the testing steps with `vagrant up`. It creates a local virtual machine.

**Comment by mkllnk** [comment]
> I had to add this ansible_python_interpreter=/usr/bin/python3

Did you run setup.yml first? That one links `python` so that Ansible works.
